scripts/models/repo.js

```diff
@@ -8,30 +8,27 @@
 
   reposObj.requestRepos = function(callback) {
     // NOTE: refactor this request into an $.ajax call
-    $.when(
-     $.ajax({
-       url: 'https://api.github.com/user/repos' +
-            '?per_page=10' +
-            '&sort=updated',
-       type: 'GET',
-       headers: { 'Authorization': 'token ' + githubToken },
-       success: function(data) {
-         // NOTE: since the 'data' paramter comes back as an
-         // array of objects, we can reassign allRepos below.
-         reposObj.allRepos = data;
-       }
-     }),
-     $.ajax({
-       url: 'https://api.github.com/users/patci/followers' +
-            '?per_page=5' +
-            '&sort=updated',
-       type: 'GET',
-       headers: { 'Authorization': 'token ' + githubToken },
-       success: function(data) {
-         reposObj.followers = data;
-       }
-     })
-    ).done(callback);
+    $.get(
```

Comment:
You want to leave the $.when, because it allows you to pass it multiple "deferred objects," or what comes back from a jQuery AJAX call. You would be making 2 GET requests separated by a comma, as they are arguments to $.get.  You can then chain a .done to the .when that will only run once ALL of the asynchronous events inside of the .when have finished running. That's where you would be calling your callback.